app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,10 @@
+from app.players.player import Player
+
+
+class Dwarf(Player):

```

Comment:
![image](https://user-images.githubusercontent.com/103201010/218766043-d38b8e05-36f9-4b0e-b68e-02cabfef7c91.png)
fix this